# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/net-tools/nameif-linenum.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- net-tools-1.60/nameif.c.orig	2003-08-16 12:16:30.000000000 +0200
+++ net-tools-1.60/nameif.c	2003-08-16 12:12:55.000000000 +0200
@@ -178,7 +178,7 @@
 			continue; 
 		n = strcspn(p, " \t"); 
 		if (n > IFNAMSIZ) 
-			complain(_("interface name too long at line %d"), line);  
+			complain(_("interface name too long at line %d"), linenum);  
 		memcpy(ch->ifname, p, n); 
 		ch->ifname[n] = 0; 
 		p += n; 
